source/releasenotes/2024-05-07-wordpress-6-5-3-update.md

```diff
@@ -0,0 +1,13 @@
+---
+title: WordPress 6.5.3 maintenance update
```

Comment:
```suggestion
title: WordPress 6.5.3 Maintenance Update
```

Reply:
last several updates have not used title case